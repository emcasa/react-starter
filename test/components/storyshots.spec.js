import path from 'path'
import initStoryshots, {
  multiSnapshotWithOptions
} from '@storybook/addon-storyshots'

class Stories2SnapsConverter {
  getSnapshotFileName(context) {
    const {kind} = context
    return path.resolve('test/components/__snapshots__', kind) + '.snap'
  }
}

initStoryshots({
  test: multiSnapshotWithOptions(),
  stories2snapsConverter: new Stories2SnapsConverter()
})
