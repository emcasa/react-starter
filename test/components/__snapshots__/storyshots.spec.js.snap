// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots auth/Login Login 1`] = `
<div
  className="sc-bdVaJa eRfVYJ"
  height="100%"
  style={
    Object {
      "display": "flex",
      "flexDirection": "column",
    }
  }
>
  <div
    className="sc-bwzfXH fywJtj"
    height="100%"
    width="100%"
  >
    <div
      className="sc-bwzfXH budnlT"
    >
      <p
        className="sc-bxivhb jePltJ"
        color="dark"
        fontSize="large"
      >
        Login with AccountKit
      </p>
    </div>
    <div
      className="sc-bwzfXH bMFILz"
    >
      <button
        className="sc-htpNat kdlsFY"
        fontSize="default"
        height="medium"
      >
        Login via Email
      </button>
    </div>
    <div
      className="sc-bwzfXH bMFILz"
    >
      <button
        className="sc-htpNat kdlsFY"
        fontSize="default"
        height="medium"
      >
        Login via SMS
      </button>
    </div>
  </div>
</div>
`;
